src/pages/workspace/WorkspaceMembersPage.tsx

```diff
@@ -216,8 +230,50 @@ function WorkspaceMembersPage({personalDetails, route, policy, currentUserPerson
 
         // Remove the admin from the list
         const accountIDsToRemove = session?.accountID ? selectedEmployees.filter((id) => id !== session.accountID) : selectedEmployees;
+
+        // Check if any of the account IDs are approvers
+        const hasApprovers = accountIDsToRemove.some((accountID) => Member.isApprover(policy, accountID));
+
+        if (!hasApprovers) {
+            setSelectedEmployees([]);
+            setRemoveMembersConfirmModalVisible(false);
+
+            InteractionManager.runAfterInteractions(() => {
+                Member.removeMembers(accountIDsToRemove, route.params.policyID);
+            });
+
+            return;
+        }
+
+        const ownerEmail = ownerDetails.login;
+
+        accountIDsToRemove.forEach((accountID) => {
+            const removedApprover = personalDetails?.[accountID];
+
+            if (!removedApprover?.login || !ownerEmail) {
+                return;
+            }
+
+            const updatedWorkflows = updateWorkflowDataOnApproverRemoval({
+                approvalWorkflows,
+                removedApprover,
+                ownerDetails,
+            });
+
+            updatedWorkflows.forEach((workflow) => {
+                if (workflow?.removeApprovalWorkflow) {
+                    const {removeApprovalWorkflow, ...updatedWorkflow} = workflow;
+
+                    Workflow.removeApprovalWorkflow(policyID, updatedWorkflow);
+                } else {
+                    Workflow.updateApprovalWorkflow(policyID, workflow, [], []);
+                }
+            });
+        });
```

Comment:
Minor change to avoid repetition
```suggestion
        if (hasApprovers) {
            const ownerEmail = ownerDetails.login;

            accountIDsToRemove.forEach((accountID) => {
                const removedApprover = personalDetails?.[accountID];

                if (!removedApprover?.login || !ownerEmail) {
                    return;
                }

                const updatedWorkflows = updateWorkflowDataOnApproverRemoval({
                    approvalWorkflows,
                    removedApprover,
                    ownerDetails,
                });

                updatedWorkflows.forEach((workflow) => {
                    if (workflow?.removeApprovalWorkflow) {
                        const { removeApprovalWorkflow, ...updatedWorkflow } = workflow;

                        Workflow.removeApprovalWorkflow(policyID, updatedWorkflow);
                    } else {
                        Workflow.updateApprovalWorkflow(policyID, workflow, [], []);
                    }
                });
            });
        }
```

Reply:
it's done